hw/ip/prim/doc/prim_keccak.md

```diff
@@ -0,0 +1,79 @@
+----
+title: "Primitive Component: Keccak permutation"
+----
+
+# Overview
+
+`prim_keccak` is a single round implementation of the permutation stage in [SHA3 algorithm][fibs-pub-202].
+Keccak primitive module assumes the Nr is less than or equal to 12 + 2L.
+It supports all combinations of the data width described in the [spec][fibs-pub-202].
+The implementation is not hardened against side-channel or fault injection attacks.
```

Comment:
not _currently_ hardened perhaps?

Reply:
I would say "This implementation" ... then we can have a prim_keccak_{something} in parallel that has countermeasures added, potentially?  That way the prim_keccak (unhardened) can be maintained separately and probably left alone once we have stabilized it.

Reply:
Thanks! Revised